taskcluster/mozillavpn_taskgraph/transforms/beetmover.py

```diff
@@ -66,27 +111,33 @@ def add_beetmover_worker_config(config, tasks):
                     },
                 }
             )
-
+        attributes = {
+            **task["attributes"],
+            "shipping-phase": shipping_phase,
+        }
         worker = {
             "upstream-artifacts": upstream_artifacts,
             "bucket": bucket,
-            "action": "push-to-candidates",
+            "action": "push-to-candidates"
+            if shipping_phase.endswith("addons") or shipping_phase.startswith("promote")
```

Comment:
I think you mentioned this in our chat, but remind me why we need to use `push-to-candidates` for the `ship-addons` phase?

Reply:
I switched to `direct-push-to-bucket`

Now I am thinking of landing something I can use to copy the candidate artifacts in beetmover properly